File: List6/Task2/Java/CovidSum.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class CovidSum {
    public static int covidSum(String country, int month) throws FileNotFoundException {
        int covidCases = 0;
        ArrayList<ArrayList<String>> listOfLists = new ArrayList<>();
        File file = new File("Covid.txt");
        Scanner sc = new Scanner(file);
        sc.nextLine();
        while (sc.hasNextLine()) {
            String line = sc.nextLine();
            ArrayList<String> covidDate = new ArrayList<>(List.of(line.split("\t")));
            listOfLists.add(covidDate);
        }

        for (ArrayList<String> listOfList : listOfLists) {
            if (month == Integer.parseInt(listOfList.get(2)) && listOfList.get(6).equals(country)) {
                covidCases = covidCases + Integer.parseInt(listOfList.get(4));
            }
        }
        return covidCases;
    }
}

File: List6/Task2/Python/CovidSum.py
import pandas as pd
from sys import argv


def covid_sum():
    covid_cases = 0
    country = argv[1]
    month = int(argv[2])
    data = pd.read_csv('Covid.txt', sep='\t')
    for index, row in data.iterrows():
        if month == int(row[2]) and (country == row[6]):
            covid_cases = covid_cases + row[4]

    return covid_cases


if __name__ == '__main__':
    print(covid_sum())
